ci/rpm7/build.sh

```diff
@@ -291,22 +290,24 @@ if [ -n "${test}" ] && [ "${stage}" = "compliance" ]; then
     if [ $? -ne 0 ]; then status=false; fi
 
     make style
-    if [ "$(cat LINT | grep 'Total errors found' | awk '{ print $4 }')" != "0" ]; then  status=false; fi
+    if [ $? -ne 0 ]; then status=false; else status=true; fi
```

Comment:
Not correct, make style always return 0, that is why makefile with targets (style) replaced the original one. Should be rollback or original makefile and backend function should be changed.

Reply:
You can do the following test:

```
$ git checkout hardening/just_playing_with_travis
$ make style
./scripts/style_check_in_makefile.sh
style_check.sh: [src/lib/serviceRoutinesV2]: found no mayor errors (4 minor errors) in 5796 lines of source code in 80 files (.27% style-guide-incompatibilities)
style_check.sh: [src/lib/logSummary]: found no mayor errors (0 minor errors) in 229 lines of source code in 2 files (0% style-guide-incompatibilities)
style_check.sh: [src/lib/jsonParseV2]: found no mayor errors (3 minor errors) in 6104 lines of source code in 46 files (.04% style-guide-incompatibilities)
style_check.sh: [src/lib/apiTypesV2]: found no mayor errors (11 minor errors) in 2571 lines of source code in 24 files (.42% style-guide-incompatibilities)
style_check.sh: found 136 errors (1 mayor, 134 minor errors) in 19497 lines of source code in 66 files (.69% style-guide-incompatibilities)
Lint Errors:

0043 errors of category 'Lines should be <= 120 characters long'
0001 errors of category 'Line ends in whitespace'
0091 errors of category 'Use int16/int64/etc'
....
makefile:262: recipe for target 'style' failed
make: *** [style] Error 1
fermin@neodeb:~/src/fiware-orion$ echo $?
2
```

As you can see `make style` return not always return 0. In this case (a fail) it return 2.

Thus, I'm afraid I'm not getting your point sorry... but the fact is that CI needs fixing with regards with compliance job. Previous to the change in this PR, travis marks as success the compliance job and this is not correct, as this job should be marked as fail (as there is an style fail in the code). If my fix is not correct, please propose an alternative fix.

(In fact, my original implementation wasn't fully correct, as a success style check may shadow possible fails in the previous stages, e.g. file compliance. The new commit db824f69cc68ad15519a02326fd68f40ef94e3cc should fix this).